GoogleForCreators/web-stories-wp
# Floating menu: Hide on drag/resize/rotate

## Context
A requested feature that came out of bug bash after using the floating menu.


## Summary
Hides the floating menu when transforming an element:

https://user-images.githubusercontent.com/35983235/159749458-5e312b5d-69d2-4ecd-a0e5-357c377c2121.mp4




## Relevant Technical Choices
Just went with hiding with opacity vs un-mounting/remounting for perf on selection.


## To-do



## User-facing changes
Floating menu now hides when transforming elements. this includes:
- resize
- translate
- rotate


## Testing Instructions






This PR can be tested by following these steps:

1. Open a story
2. Add an element
3. Select the element and see that the floating menu appears
4. See that the floating menu disappears with the following actions. then reappears when complete:
- resize
- drag to move
- rotate


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11014
